cookbooks/aws-parallelcluster-environment/kitchen.environment-config.yml

```diff
@@ -33,6 +33,33 @@ suites:
         scheduler: slurm
         cw_logging_enabled: "true"
         log_group_name: test
+  - name: directory_service
+    # FIXME: breaks on Docker
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-environment::directory_service]
+    verifier:
+      controls:
+        - /tag:config_sssd_configured_correctly/
```

Comment:
The control can not be tagged with `tag:config` because it relies on attributes we set in this specific test. The control will fail in scope of `validate_config` test.